configure.ac

```diff
@@ -198,13 +198,6 @@ AS_IF([test "x$enable_manpages" = "xyes"], [
 	])])
 AM_CONDITIONAL([BUILD_MANPAGES], [test "x$enable_manpages" = "xyes"])
 
-AC_ARG_ENABLE([logging],
```

Comment:
```
If one is to switch it off, then the libkmod log API becomes
non-functional. In addition, the tools' syslog facility also does
nothing...
```

which is fine for embedded use cases relying only on return codes.

Reply:
I actually don't think there's any good reason to use the busybox variant. And I'm aware of my very biased view :)

If you look at e.g. yocto and buildroot, they both allow you to select kmod.  Android uses kmod (it used not to in ~2014, when I a talk in an Android conf about that)

If your init of choice is systemd, then it's a requirement. A decade ago I was using systemd on embedded systems... Just found a video from one of my previous bosses.... https://www.youtube.com/watch?v=ERS1OSOTGpg&ab_channel=TheLinuxFoundation - you can find a bunch (more recent ones) if you google "embedded" and "systemd".

Reply:
To their defense, kmod 30 is listed as latest release on GitHub. All others afterwards are just tags. ;)

Of course this can't be the reason, but is there anything kmod repository can do about?